paddle/fluid/platform/device/gpu/cuda/cuda_profiler.cc

```diff
@@ -20,6 +20,7 @@ namespace platform {
 void CudaProfilerInit(const std::string& output_file,
                       const std::string& output_mode,
                       const std::string& config_file) {
+#if CUDA_VERSION < 11000
```

Comment:
Is there no need for profiler inition once cuda version was beyond cuda11.0? The pr-description is all about cuda12.0, will this change affect version between cuda11.0 and cuda12.0, such as cuda11.2 ?

Reply:
`cudaProfilerInitialize` is marked as deprecated from cuda11.0, and it is removed after cuda12.0. So this change will not affect the functionaltiy of cuda 11.x.
https://docs.nvidia.com/cuda/archive/11.0/cuda-runtime-api/group__CUDART__PROFILER__DEPRECATED.html